src/pages/en/guides/markdown-content.md

```diff
@@ -357,6 +357,24 @@ const { Content } = Astro.props.post
   <Content/>
 </article>
 ```
+## Fetching Remote Markdown
+
+Astro was designed to render Markdown to HTML as a part of building your website. However, there may be certain cases where you need to fetch Markdown from a remote source at runtime, after the site has been built. For example, you may want to fetch and render Markdown from a remote API every time a user makes a request to your website.
+
+**Astro does not include a runtime Markdown parser!** To render remote Markdown to HTML at runtime, you will need to install and configure your own Markdown parser. This **will not** inherit from any of Astro's built-in Markdown and MDX settings that you have configured. Be sure that you understand these limitations before implementing this in your project. 
+
+```astro
```

Comment:
```suggestion
```astro title="src/pages/markdown-cheatsheet.astro"
```
@FredKSchott took the liberty of simply moving this up near "Importing Markdown" and renaming it "Fetching Remote Markdown" to a) be consistent with the -ing of the other nearby headings and b) bring it closer to the other section about injecting Markdown onto a page (vs the `.md` file being its own page).

Otherwise, I think I'd probably just suggest a file name title for this, so that it's clear this is happening in an `.astro` component, but good to have mention of this in the docs!